CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ $ pip install -U blue
 $ blue .
 ```
 
-You can check all this by running ``tox`` from within the Falcon project directory. Your environment must be based on CPython 3.8, 3.10 or 3.11:
+You can check all this by running ``tox`` from within the Falcon project directory. Your environment must be based on CPython 3.8, 3.10, 3.11 or 3.12:
```

Comment:
Maybe it's ok to just use 3.8-3.12

Reply:
3.9 doesn't work well (explained below), so I think it is better to keep it clear. Moreover, it was like that before this PR.

Reply:
oh, forgot about that 3.9 issue